packages/components/src/Actions/Action/Action.component.js

```diff
@@ -101,7 +103,11 @@ function Action(props) {
 	});
 
 	const buttonContent = getContent(props);
-
+	if (available === false) {
+		buttonProps.className = classnames({
+			hidden: available === false,
+		}, buttonProps.className || '');
```

Comment:
`available === false` --> `!available` ?
`buttonProps.className || ''` --> `buttonProps.className`, the falsy value is managed by `classNames()`